nbd/src/util.h

```diff
@@ -60,6 +60,8 @@ extern int check_block_size(int nbd_index, uint64_t expected_size);
 extern int check_device_size(int nbd_index, uint64_t expected_size);
 // 如果当前系统还未加载nbd模块，则进行加载；如果已经加载，则不作任何操作
 extern int load_module(NBDConfig *cfg);
+// Check the mount status and judge whether it can be unmap
+extern int check_mount_status(const NBDConfig *cfg);
```

Comment:
`extern` in here is redundant, and maybe `check_dev_can_unmap` is more obvious

Reply:
> check_dev_can_unmap

done